logs/app.log

```diff
@@ -0,0 +1,12 @@
+2023-06-07 11:47:43,940 DEBUG mate.academy.service.AuthenticationServiceImpl:12 login method was called. Params: login = bob

```

Comment:
It is no sense to add this `app.log` file to  the`.gitignore` and after that push it to the repo.
please remove this file from git and persuade that it will not be added to the new git commits.